config/static_provider.go

```diff
@@ -20,13 +20,7 @@
 
 package config
 
-import (
-	"fmt"
-	"sync"
-)
-
 type staticProvider struct {
-	sync.RWMutex
```

Comment:
Added a related comment, a normal copy would require lots of reflection tricks, which we can add later.